recipes/pylev/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pylev" %}
+{% set version = "1.3.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "063910098161199b81e453025653ec53556c1be7165a9b7c50be2f4d57eae1c3" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - pylev
+  requires:
+    - pylev
```

Comment:
This is not needed.  This is only needed if specific requirements are needed during testing (e.g., `pytest`, `nose`, etc..)